src/components/auth/auth.service.ts

```diff
@@ -184,6 +188,93 @@ export class AuthService {
     return session;
   }
 
+  async forget(input: UserEmailInput): Promise<boolean> {
+    const token = this.encodeJWT();
+    const email = input.email;
+
+    await this.db
+      .query()
+      .raw(
+        `
+      CREATE(et:EmailToken{value:$value, token: $token, createdOn:datetime()})
+      RETURN et as emailToken
+      `,
+        {
+          $value: email,
+          $token: token,
+        }
+      )
+      .first();
+    const params = {
+      Destination: { ToAddresses: [email] },
+      Message: {
+        Body: {
+          Html: {
+            Charset: 'UTF-8',
+            Data: `<html><body><p>This is your secret login code:</p>
+                          <a href="http://localhost:3333/auth/reset?token=${token}">Go to Login</a></body></html>`,
+          },
+          Text: {
+            Charset: 'UTF-8',
+            Data: `http://localhost:3333/auth/reset?token=${token}`,
+          },
+        },
+        Subject: {
+          Charset: 'UTF-8',
+          Data: 'Forget Password',
+        },
+      },
+      Source: this.env.string('SOURCE_EMAIL').optional('core-field'),
```

Comment:
`EnvironmentService` is meant to be a private helper service for `ConfigService` (as noted in the class docs). This is also why it's not exported from the `CoreModule`.

Can you move this to the `ConfigService`? Something like:
```ts
class ConfigService {
  emailFrom = this.env.string('EMAIL_FROM').optional('noreply@cordfield.com');
}
```
The reason for this is to prevent using "magic strings" all over the codebase and centralizes it all in one spot.